src/views/List/List.jsx

```diff
@@ -164,7 +192,12 @@ export default function List({ token }) {
                     id={name}
                     name={listItem.id}
                   />{' '}
-                  <label htmlFor={name}>{name}</label>
+                  <label htmlFor={name}>
+                    {name}{' '}
+                    <small className="badge">
```

Comment:
I learned about <small> tonight too! 🌱

From my reading, it seems like <.small.> and <.strong.> are opposites and wouldn't be applied to the same text item because they convey different info semantically: small is meant to draw attention away from that text and strong is meant to draw a user's attention to that text if this is the wrong interpretation let me know but that's where my thoughts are 

I would use one or the other but not both -- the badge could be a small accent piece or a big bold label

This is obviously a very small kind of nitpicky thing I just really like semantic HTML thank you for showing me <.small.> ! Semantic HTML is very fun to think about and dissect